pkg/crc/machine/libvirt/constants.go

```diff
@@ -6,10 +6,6 @@ const (
 	DefaultCacheMode = "default"
 	DefaultIOMode    = "threads"
 
-	// Static reesources
-	PoolName = "crc"
```

Comment:
These weren't used in this repo or the https://github.com/code-ready/machine-driver-libvirt repo

Reply:
not used anymore...